src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,50 @@
 
 class Animal {
   // write your code here
+  constructor(name, health) {
+    this.name = name;
+    this.health = 100;
+
+    Animal.alive.push(this);
+  }
+  static alive = [];
+
+  die() {
+    const index = Animal.alive.indexOf(this);
+
+    if (index !== -1) {
+      Animal.alive.splice(index, 1);
```

Comment:
Consider using the `filter` method to remove dead animals from the `Animal.alive` array, as it is preferred according to the checklist.